string studentId
string fullName
string grade
float64 scoreA
float64 scoreB
float64 scoreC

